src/components/blockHelpers.tsx

```diff
@@ -365,7 +366,11 @@ export const ContentWrapperStyled = styled('div')<{
     padding: '0 ' + CONTENT_GUTTER_MOBILE,
   },
 
-  ...(brandPivot ? CONTENT_MAX_WIDTH : CONTENT_MAX_WIDTH_DEPRECATED),
+  ...(brandPivot
+    ? fullWidth
+      ? SITE_MAX_WIDTH
+      : CONTENT_MAX_WIDTH
+    : CONTENT_MAX_WIDTH_DEPRECATED),
```

Comment:
could we maybe move this to a function and name it? Ithink nested ternaries get really hard to read